src/SDKs/Azure.Core/data-plane/Azure.Core/Azure.Core.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the implementation of the Azure Client Pipeline</Description>
+    <AssemblyTitle>Microsoft Azure Client Pipeline</AssemblyTitle>
+    <Version>1.0.0-preview.1</Version>
+    <PackageTags>Microsoft Azure Client Pipeline</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+      ]]>
+    </PackageReleaseNotes>
+
+    <!-- This is a workaorund until https://github.com/Azure/azure-sdk-for-net/issues/5214 is addressed -->
+    <RequiredTargetFrameworks>net461;netstandard2.0</RequiredTargetFrameworks>
+    
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <LangVersion>7.2</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Buffers" Version="4.5.0" />
+    <PackageReference Include="System.Memory" Version="4.5.1" />
+    <PackageReference Include="System.Text.Primitives" Version="0.1.0-preview1-180216-4" />
```

Comment:
This should not be here. Do you have an old snapshot from labs? 
See https://github.com/Azure/azure-sdk-for-net-lab/blob/master/Core/Azure.Core/Azure.Core.csproj

Reply:
Yes -- this was a slighly older version of the code -- this is now fixed.